core/src/test/java/io/substrait/relation/VirtualTableScanTest.java

```diff
@@ -0,0 +1,52 @@
+package io.substrait.relation;
+
+import static io.substrait.expression.ExpressionCreator.*;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Could you import these explicitly instead of with `*`.

Reply:
Done! [86a19aa](https://github.com/substrait-io/substrait-java/pull/255/commits/86a19aae9697c89f16f335ffd5994d0328b6fae8)